hdr/sqlite_modern_cpp.h

```diff
@@ -173,7 +174,12 @@ namespace sqlite {
       return sqlite3_sql(_stmt.get());
     }
 
-		void used(bool state) { execution_started = state; }
+		void used(bool state) {
+			if(execution_started == true && state == true) {
+				throw sqlite_exception("Already used statement executed again! Please reset() first!",sql());
```

Comment:
Maybe in this situation `assert` would be a better fit, because this can only be triggered by coding errors.